### PR TITLE
Update the order search endpoint for statuses

### PR DESCRIPTION
**Pull request by Gidgidonihah**
Alter the params to match the upcoming change to status requests.

Relies upon https://github.com/DobaTech/thanos/issues/687 being released.

### CODE REVIEW

**Comment by baileymatthewr** [comment]
@Gidgidonihah: gh-pages now has this stuff added to it.

File: _ordering/get_order_list.md [hunk 1]
```diff
@@ -362,8 +363,11 @@ limit
 sort
 : (object) The Sort object contains a Key to sort on and a Direction (dir) to sort in
 
-status
-: (string) The Status of the Orders you would like to see returned
+line_item_status
+: (array) One or more line item statuses. Will include orders with statuses based upon that `status_conjunction`. Possible Values are: `unallocated`, `allocated`, `rejected`, `has_tracking`, `backordered`
```
Comment (edje-jeter) [inline _ordering/get_order_list.md]: I think the conjunction sentence is confusing. Maybe:
(array) One or more line item statuses: `unallocated`, `allocated`, `has_tracking`, `backordered`, `rejected`. The response will contain orders with statuses that match the provided `line_item_status` (as combined based on `status_conjunction`).

File: _ordering/get_order_list.md [hunk 2]
```diff
@@ -362,8 +363,11 @@ limit
...
+
+status_conjunction
+: (string) Indicates how the line_item_status relates to the order response. Possible values are: `and`, `or`, `all`. Default: `or`. `And` will return all orders that have at least 1 line item matching each supplied status. `or` will return all orders that have at least one line item matching any of the supplied statuses. `only` will return orders that have ALL line items matching the supplied status. Only one line_item_status should be supplied when using `only`.
```
Comment (edje-jeter) [inline _ordering/get_order_list.md]: The possible values are not quite right and I think 'relates to order response' is unclear. Maybe:

> (string) Determines whether search for `line_item_status` returns orders with a union, intersection, or complement of statuses. Possible values: `and`, `or`, `only`. Default: `or`. The conjunction determines whether we return every order that contains all (`and`), at least one (`or`), or only (`only`) the provided line-item-status(es). The conjunction `only` must have exactly one `line_item_status`.

**Comment by baileymatthewr** [comment]
@Gidgidonihah : Can you please merge and release this?

**Pull request merged**